tests/integration/conftest.py

```diff
@@ -26,21 +27,27 @@
 from pytest_operator.plugin import OpsTest
 
 from charm_state import (
+    LABELS_CONFIG_NAME,
     OPENSTACK_CLOUDS_YAML_CONFIG_NAME,
     OPENSTACK_FLAVOR_CONFIG_NAME,
     OPENSTACK_NETWORK_CONFIG_NAME,
     PATH_CONFIG_NAME,
+    USE_APROXY_CONFIG_NAME,
     VIRTUAL_MACHINES_CONFIG_NAME,
 )
 from github_client import GithubClient
-from tests.integration.helpers import (
+from tests.integration.helpers.common import (
+    InstanceHelper,
     deploy_github_runner_charm,
-    ensure_charm_has_runner,
     reconcile,
     wait_for,
 )
+from tests.integration.helpers.lxd import LXDInstanceHelper, ensure_charm_has_runner
+from tests.integration.helpers.openstack import OpenStackInstanceHelper
 from tests.status_name import ACTIVE
 
+nest_asyncio.apply()
```

Comment:
Would it be good to add a short comment on what is this for?
Maybe a link to docs?